lib/rubocop/cop/factory_bot/create_list.rb

```diff
@@ -32,26 +34,35 @@ module FactoryBot
       #   create_list :user, 3
       #
       #   # good
-      #   3.times { create :user }
+      #   3.times.map { create :user }
       #
       class CreateList < ::RuboCop::Cop::Base
         extend AutoCorrector
         include ConfigurableEnforcedStyle
         include RuboCop::FactoryBot::Language
 
         MSG_CREATE_LIST = 'Prefer create_list.'
-        MSG_N_TIMES = 'Prefer %<number>s.times.'
+        MSG_N_TIMES = 'Prefer %<number>s.times.map.'
         RESTRICT_ON_SEND = %i[create_list].freeze
 
-        # @!method array_new_or_n_times_block?(node)
-        def_node_matcher :array_new_or_n_times_block?, <<~PATTERN
-          (block
-            {
-              (send (const {nil? | cbase} :Array) :new (int _)) |
-              (send (int _) :times)
-            }
-            ...
-          )
+        # @!method repetition_block?(node)
+        def_node_matcher :repetition_block?, <<-PATTERN
+          (block {#array_new? #n_times? #n_times_map?} ...)
```

Comment:
memo) I know this is a different topic than this PR, but should we support numblock?

Reply:
It might be, as those blocks yield sensible arguments:
```ruby
Array.new(3) do |index|
  create(:user, some_attr: index)
end
```
I can't imagine why would anyone want to use `_1` here, but it's a matter of taste and commitment to trends 😄

Reply:
Well, normally we can just use Faker, but sometimes I see code that tries to be unique in the index like `uniq_name_#{_1}`, should that be another cop to offense?

Reply:
I was wondering if there could be a use case for this kind of thing. This could be a real edge case that we may not have to deal with 😅